sources/web/datalab/polymer/components/file-browser/file-browser.ts

```diff
@@ -915,7 +921,11 @@ class FileBrowserElement extends Polymer.Element implements DatalabPageElement {
     // Always add the root file to the beginning.
     const root = await this._fileManager.getRootFile();
     this._pathHistory.unshift(root);
-    this._pathHistoryIndex = this._pathHistory.length - 1;
+    if (this._pathHistoryIndex === this._pathHistory.length - 1) {
+      this._pathHistoryIndexChanged();
```

Comment:
I'm not sure I understand the logic here. Do we want the callback to fire all the time?

Reply:
Yes. For drive, the history length is always 1, but we're changing it and we want the code that reloads the file list to get executed. That code is triggered on a change to pathHistoryIndex, so even when pathHistoryIndex is not changing, we still want to call that method here.